### PR TITLE
Updates utils with a few additional date helper functions

### PR DESCRIPTION
**Pull request by ngodavidhuy**

### CODE REVIEW

File: client/src/utils/calendar-helpers.js [hunk 1]
```diff
@@ -80,9 +90,50 @@ const generateCalendarMonth = (monthIndex, year) => {
   return null;
 };
 
+const calculateDateTo = (dateFrom) => {
+  let dateTo = new Date(dateFrom.getFullYear(), dateFrom.getMonth(), dateFrom.getDate() + 61);
+  return dateTo;
+}
+const formatDate = (date) => {
```
Comment (e553y) [inline client/src/utils/calendar-helpers.js]:
```suggestion
 const formatDate = (date) => {
    return date.toLocaleDateString()
 }
```

File: client/src/utils/calendar-helpers.js [hunk 2]
```diff
@@ -80,9 +90,50 @@ const generateCalendarMonth = (monthIndex, year) => {
...
+  if (date instanceof Date) {
+    let month = date.getMonth();
+    let day = date.getDate();
+    let year = date.getFullYear();
+    
+    return `${month + 1}/${day}/${year}`;
```
Comment (e553y) [inline client/src/utils/calendar-helpers.js]: I think date.toLocaleDateString() can help here

Comment (e553y) [reply client/src/utils/calendar-helpers.js]: maybe for next refactor

Comment (ngodavidhuy) [reply client/src/utils/calendar-helpers.js]: Will do!

**Pull request merged**